src/main/java/seedu/address/model/person/Address.java

```diff
@@ -11,7 +11,9 @@
  */
 public class Address {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, "
+            + "and it should not be blank."
+            + "If you want to not specify the address, leave out a/ADDRESS entirely.";
```

Comment:
Is this statement correct for `edit` command?

Reply:
This one will not be shown for edit command as address accept all.